examples/calculations/Sounding_Calculations.py

```diff
@@ -0,0 +1,210 @@
+# Copyright (c) 2015-2022 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""
+=============================
+Sounding Calculation Examples
+=============================
+
+Use functions the `metpy.calc` to perform a number of calculations using sounding data.
+
+The code below uses example data to perform many sounding calculations for a severe weather
+event on May 22, 2011 from the Norman, OK sounding.
+"""
+import numpy as np
+import pandas as pd
+
+import metpy.calc as mpcalc
+from metpy.cbook import get_test_data
+from metpy.units import units
+
+###########################################
+# Effective Shear Algorithm for use in Supercell Composite Calculation
+
+
+def effective_layer(p, t, td, h, height_layer=False):
```

Comment:
We should consider just adding this to `metpy.calc` if we think it's the right way to do this.

Reply:
Yes, although maybe start with this in the example to give it a soft test and mark to add in the next release?

Reply:
Fair enough.